src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -60,7 +60,7 @@ export default class DetailsBody extends React.Component<Props> {
     const discoverQuery: NewQuery = {
       id: undefined,
       name: (incident && incident.title) || '',
-      fields: ['issue', 'count(id)', 'count_unique(user.id)'],
+      fields: ['issue', 'count(id)', 'count_unique(user.id)', 'title'],
```

Comment:
Is the order right? Should it be 1st or 2nd?

Reply:
Hmm not sure. I made it 2nd. Gonna check with Adhi

Reply:
Adhi likes 2nd.